modules/distr-tracing-config-security-ossm-cli.adoc

```diff
@@ -0,0 +1,93 @@
+////
+This module included in the following assemblies:
+service_mesh/v2x/ossm-reference-jaeger.adoc
+////
+:_content-type: PROCEDURE
+[id="distr-tracing-config-security-ossm-cli_{context}"]
+= Configuring distributed tracing security for service mesh from the command line
+
+You can modify the Jaeger resource to configure {JaegerShortName} security for use with {SMproductShortName} from the command line using the `oc` utility.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.  If you use {product-dedicated}, you must have an account with the `dedicated-admin` role.
+* The {SMProductName} Operator must be installed.
+* The `ServiceMeshControlPlane` deployed to the cluster.
+* You have access to the OpenShift CLI (oc) that matches your OpenShift Container Platform version.
+
+.Procedure
+
+. Log in to the {product-title} CLI as a user with the `cluster-admin` role. If you use {product-dedicated}, you must have an account with the `dedicated-admin` role.
```

Comment:
```suggestion
. To log in to the {product-title} CLI as a user with the `cluster-admin` role, enter the following command. If you use {product-dedicated}, you must have an account with the `dedicated-admin` role.
```
Optional: See the [repo guidelines](https://www.ibm.com/docs/en/ibm-style?topic=elements-ui#verbs-to-use) for lead-in sentence patterns applicable in most cases.